library/src/main/java/com/google/android/material/motion/runtime/Scheduler.java

```diff
@@ -172,6 +172,33 @@ public void addPlan(Plan plan, Object target) {
     getTargetScope(target).commitPlan(planInfo);
   }
 
+  /**
+   * Adds a {@link NamedPlan} to this scheduler.
+   * When this method is invoked, a {@link NamedPlan} with the same name and target
+   * is removed from the scheduler before the plan is eventually added.
```

Comment:
it looks like the latest commit does include your new documentation. Github PR just has a confusing UI when you click on a comment that was made on a previous commit.